docs/modules/ROOT/partials/install/bootstrap-nodes.adoc

```diff
@@ -146,15 +201,7 @@ done && echo -e "\nCilium config updated"
 kubectl -n cilium rollout restart ds/cilium
 ----
 
-. Review and merge the LB hieradata MR (listed in Terraform output `hieradata_mr`) and run Puppet on the LBs after the deploy job has completed
```

Comment:
This step can't be deleted unconditionally, it's required to ensure that the Puppet LBs have a correct configuration for the ingress backends.

Additionally, we probably need a replacement step for cloudscale which somehow sets the infra node IPs as ingress backends since Terraform doesn't have those IPs anymore.

Reply:
It's not deleted, I just shuffled it around. It should be further up now.